lib/health_checking/health_checks/github_repo_webhooks.rb

```diff
@@ -0,0 +1,50 @@
+module HealthChecker
+  module HealthChecks
+    class GitHubRepoWebhooks
+      include HealthCheck
+
+      name 'github_repo_webhooks'
+      weight :warning
+      authorization :collaborator
+      message "Click <a href='/some/place'> Here </a> to fix"
+
+      ## deps
+      # {
+      #   repo: repo object,
+      #   authorization: :all, :collaborator or :admin
+      #   logged_in: bool
+      # }
+      ##
+      
+      def self.perform(deps)
```

Comment:
if we force the checks to be instances ie;

```
def perform(deps)
    #=> stuffs
end
```
They will conform to the same constraints as ActiveJob jobs, it gives us the flexibility to offload them to a separate process 

Also makes things easy to multi-thread